tests/fixtures.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from copy import deepcopy
-
 import pytest
-
+from sys import version_info
 from pydash._compat import iteritems
 
+if version_info[0] > 2:
```

Comment:
Python 2.7 actually raises a `SyntaxError` for a file containing type annotations, which is why I don't believe the `pytest.mark.skipif` will work and instead went with the dynamic import & changed the test targets below. Wasn't sure the best way to approach this.